src/irmin-pack/unix/gc.ml

```diff
@@ -17,6 +17,310 @@
 open! Import
 include Gc_intf
 
+module Steps_timer = struct
```

Comment:
maybe these next three modules can be moved into a separate file `gc_stats.ml` for instance?